.husky/pre-push

```diff
@@ -1,4 +1,4 @@
 #!/bin/sh
 . "$(dirname "$0")/_/husky.sh"
 
-sh ./scripts/enforceVersions.sh && yarn test:unit -- --changedSince=latest
+sh ./scripts/enforceVersions.sh && yarn test:unit --changedSince=latest --passWithNoTests
```

Comment:
Check why Yarn v3 might be affecting the jest test files when they haven't been changed